packages/communication-react/review/communication-react.api.md

```diff
@@ -1435,7 +1435,7 @@ export interface MeetingCompositeProps extends BaseCompositeProps<CallCompositeI
     formFactor?: 'desktop' | 'mobile';
     // (undocumented)
     meetingAdapter: MeetingAdapter;
-    meetingCallControlOptions?: MeetingCallControlOptions;
+    meetingCallControlOptions?: boolean | MeetingCallControlOptions;
```

Comment:
Should we extract `MeetingCompositeOptions` similar to `CallCompositeOptions`?

Reply:
maybe extend it so we can pass those options directly to the calling composite from the meeting composite? I think the other properties in the calling composite will follow 'default' behavior otherwise?

Reply:
Yeah, let's do that as a follow up PR regardless.